src/hook.tsx

```diff
@@ -26,7 +26,7 @@ const useAttention = (
   const attention = useContext(AttentionContext);
   if (!attention) throw new Error('`useAttention` can only be used inside `AttentionProvider`.');
 
-  useEffect(() => {
+    useEffect(() => {
```

Comment:
This is incorrect. Can you run `bun run format` please, and copy the script from our other repos?